staking/stakingprecompile.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Staking Precompile
+description: Moonbeam Parachain Staking Ethereum Solidity Precompile Interface Demo
+---
+
+# Staking Precompile
+
+![Staking Moonbeam Banner](/images/staking/staking-stake-banner.png)
```

Comment:
I would create a different banner not to reuse the same old ones. Probably a contract (look for OpenZeppelin contracts to grab it as a template) and some tokens in it or something)